parsers/ocaml.c

```diff
@@ -1809,9 +2024,20 @@ static void localScope (vString * const ident, ocaToken what)
 static void computeModuleName ( void )
 {
 	/* in Ocaml the file name define a module.
-	 * so we define a module =)
+	 * so we define a module if the file has
+	 * things in it. =)
 	 */
 	const char *filename = getSourceFileName ();
+
+	FILE *file = fopen (filename, "r");
+	fseek (file, 0, SEEK_END);
+	long fileSize = ftell (file);
+	fclose (file);	// CLOSE!
+	if (fileSize == 0)
+	{
+		return;
+	}
```

Comment:
> What about creating the module tag together with the first tag found?

Sounds fair, though it might then mean the source position would not be (by default) on the first line/pos.  But that'd be a matter of recording the source position at the start of the file and use it later.